### ISSUE TITLE
Add a history of adresses connected to

### ISSUE THREAD

**Issue by Force-quit** [Force-quit/Chatlis#40]
Make the adress of the server to connect to a drop down so that you can connect to previously connected servers (without having to type their adresses again).

**Comment by CmD0** [comment]
Are we saving this somewhere or is it only a runtime history?

**Comment by Force-quit** [comment]
Good idea, we could save it in temp files. Qt has QStandardPaths::writableLocation so you can get the "Temp" folder's path.
